<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="NASDAQCM">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Capital Market</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="NASDAQGM">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Global MarketSM</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="NASDAQGSM">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Global Select MarketSM</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Intervial_1Day">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>28</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1d</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Intervial_1h">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1h</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Intervial_30min">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>30min</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IncDownload">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>增量下载</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">DownloadIR</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="ReDownload">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>重新下载</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">DownloadIR</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="CancelDownload">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmDownload">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>280</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QSlider" name="DownloadPeriod">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>771</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="maximum">
    <number>11</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="Intervial_15min">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>15min</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Intervial_5min">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>5min</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Intervial_1min">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>大盘</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenDLConfig">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveDLConfig">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="ShowPeriod">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>30 Day</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ETF指数</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="VPNEnable">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VPN</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="List_Favor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Favor</string>
   </property>
  </widget>
  <widget class="QTableView" name="Favor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="MulitThreadDL">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ShowPeriod_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>下载线程</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Intervial_Current">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ReConnect">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ShowPeriod_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>重传次数</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TimeOut">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ShowPeriod_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>超时设置</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Financials">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>财报</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Balance_Sheet">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>资产负载表</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Cashflow">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>现金流</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="SymbolsListProgressBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>43</number>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateTickerList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>更新清单</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="NYSEMKT">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>NYSE MKT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="NYSE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>NYSE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="NYSEARCA">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>NYSE ARCA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IEXG">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>IEXG</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="BATS">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>BATS</string>
   </property>
  </widget>
  <widget class="QLabel" name="ShowPeriod_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NASDAQ</string>
   </property>
  </widget>
  <widget class="QLabel" name="ShowPeriod_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NYSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditFavorList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑Favor</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="DownloadIR"/>
 </buttongroups>
</ui>
